operations/node_management/Add_Remove_Replace_NCNs/Boot_NCN.md

```diff
@@ -105,20 +127,119 @@ Boot a master, worker, or storage non-compute node (NCN) that is to be added to
+    ncn-mw# cray bss bootparameters list --name ${XNAME} --format=json |jq .[]> ${XNAME}.check.json
     ```
 
     * Compare the new JSON file with what was PUT to BSS.
 
     ```bash
-    ncn# diff ${XNAME}.json ${XNAME}.check.json
+    ncn-mw# diff ${XNAME}.json ${XNAME}.check.json
     ```
 
     * The files should be identical
 
-### Step 5 - For the next step please click the link for the node type you are adding.
+### Step 6 - Run NCN Personalizations using CFS as desired
+
+1. Run the following commands to list the available configurations.
+
+    ```bash
+    ncn-mw# cray cfs configurations list
+    ```
+
+    Example Output:
+
+    ```screen
+    [[results]]
+    lastUpdated = "2022-03-14T20:59:44Z"
+    name = "ncn-personalization"
+    [[results.layers]]
+    cloneUrl = "https://api-gw-service-nmn.local/vcs/cray/csm-config-management.git"
+    commit = "1dc4038615cebcfad3e8230caecc885d987e8148"
+    name = "csm-ncn-1.6.28"
+    playbook = "site.yml"
+    ```
+    
+2. Or determine the configuration applied another NCN of the same type. This example checks the configuration on `ncn-w002`.
+
+    ```bash
+    ncn-mw# cray cfs components describe $(ssh ncn-w002 cat /etc/cray/xname)
+    ```
+
+    Example Output:
+
+    ```screen
+    configurationStatus = "configured"
+    desiredConfig = "ncn-personalization"
+    enabled = true
+    errorCount = 0
+    id = "x3000c0s9b0n0"
+    [[state]]
+    cloneUrl = "https://api-gw-service-nmn.local/vcs/cray/csm-config-management.git"
+    commit = "1dc4038615cebcfad3e8230caecc885d987e8148"
+    lastUpdated = "2022-03-15T15:29:20Z"
+    playbook = "site.yml"
+    sessionName = "batcher-5e431205-a4b4-4a2e-8be3-21cf058774cc"
+    ```
+
+3. Select the appropriate configuration based on the above steps to personalize the added NCN. In this example, the `ncn-personalization` configuration is used.
+
+    ```bash
+    ncn-mw# cray cfs components update $XNAME --desired-config ncn-personalization
+    ```
+
+4. Wait for `configurationStatus` to transistion from `pending` to `configured`
 
-- [Worker node]()
-- [Storage node](../../utility_storage/Add_Ceph_Node.md)
-- [Master node]()
+    ```bash
+    ncn-mw# watch "cray cfs components describe $XNAME"
+    ```
+
+    Example Output:
+
+    ```
+    configurationStatus = "configured"
+    desiredConfig = "ncn-personalization"
+    ...
+    ```
+
+### Step 7 - Lock the management nodes
+
+Follow [How to Lock Management Single Node](../../../operations/hardware_state_manager/Lock_and_Unlock_Management_Nodes.md#to-lock-single-nodes-or-lists-of-specific-nodes-and-their-bmcs). The management nodes may be unlocked at this point. Locking the management nodes and their BMCs will prevent actions from FAS to update their firmware or CAPMC to power off or do a power reset. Doing any of these by accident will take down a management node. If the management node is a Kubernetes master or worker node, this can have serious negative effects on system operation.
```

Comment:
```suggestion
Follow the [How to Lock Management Single Node](../../../operations/hardware_state_manager/Lock_and_Unlock_Management_Nodes.md#to-lock-single-nodes-or-lists-of-specific-nodes-and-their-bmcs) procedure. The management nodes may be unlocked at this point. Locking the management nodes and their BMCs will prevent actions from FAS to update their firmware or CAPMC to power off or do a power reset. Doing any of these by accident will take down a management node. If the management node is a Kubernetes master or worker node, this can have serious negative effects on system operation.
```